Automattic/woocommerce-payments
# Add e2e test for 'Merchant / Order / Refund failure' critical flow

Closes https://github.com/Automattic/woocommerce-payments/issues/2213.

#### Changes proposed in this Pull Request

Title: This PR adds a new E2E test that covers the `Merchant / Order / Refund failure` critical flow.

Description: The test will input invalid values in these fields (highlighted in red) and then attempt to proceed with the refund:
<img width="811" alt="Screen Shot 2021-07-23 at 8 24 28 PM" src="https://user-images.githubusercontent.com/4509348/126781623-e8db6737-831e-4f9c-bd15-94877d1abf5b.png">

#### Testing instructions

1. Setup the E2E environment as usual
2. Run the newly added e2e test using this command: `npm run test:e2e ./tests/e2e/specs/merchant/merchant-orders-refund-failures.spec.js`.
3. The following test cases should pass:
    
<img width="713" alt="Screen Shot 2021-07-23 at 8 26 08 PM" src="https://user-images.githubusercontent.com/4509348/126781913-7f551afa-9a15-4a4a-bcc5-93a777491dfa.png">
